src/hooks/useURLConfigs.ts

```diff
@@ -87,8 +86,6 @@ export const useURLConfigs = (): LinksConfigs => {
     protocolStaking: linksConfigs.protocolStaking,
     stakingAndClaimingRewardsLearnMore:
       linksConfigs.stakingAndClaimingRewardsLearnMore ?? FALLBACK_URL,
-    // todo add to the link configs objects
-    vaultsLearnMore: linksConfigs.vaultsLearnMore ?? FALLBACK_URL,
```

Comment:
do we need `vaultLearnMore` since we are deprecating `vaultsLearnMore`?

Reply:
It's there!